app/components/style.tsx

```diff
@@ -16,3 +26,54 @@ export const Button = styled.button<ButtonProps>`
 export const NameShadow = styled.div<{ primary: boolean }>`
```

Comment:
Tutaj też można zdefiniować typ dla propsów NameSzadow, albo w ogóle usunąć to, bo chyba już tego nie potrzebujemy, co nie?